variables.tf

```diff
@@ -56,31 +56,42 @@ variable "domain_mapping" {
 ## IAM
 
 variable "iam" {
-  description = "(Optional) A list of IAM access to apply to the created secret."
+  description = "(Optional) A list of IAM access to apply to the created Cloud Run service."
   type        = any
   default     = []
 }
 
 variable "policy_bindings" {
-  description = "(Optional) A list of IAM policy bindings to apply to the created secret."
+  description = "(Optional) A list of IAM policy bindings to apply to the created Cloud Run service."
   type        = any
   default     = null
 }
 
+variable "computed_members_map" {
+  type        = map(string)
+  description = "(Optional) A map of members to replace in `members` to handle terraform computed values. Will be ignored when policy bindings are used."
+  default     = {}
+
+  validation {
+    condition     = alltrue([for k, v in var.computed_members_map : can(regex("^(allUsers|allAuthenticatedUsers|(user|serviceAccount|group|domain|principal|principalSet):)", v))])
```

Comment:
please update the condition [accordingly](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_service_iam)

if you copy-paste, copy from the child iam module :-)

Reply:
Thank you and nice catch. Not sure where I copied that from but apparently I didn't read it. Has been fixed.